connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -125,6 +132,39 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
         return embeddedComponentManager.getInitialParams(context)
     }
 
+    /**
+     *
+     */
+    suspend fun onPermissionRequest(context: Context, request: PermissionRequest) {
+        // we only care about camera permissions (audio/video)
+        val permissionsRequested = request.resources.filter {
+            it in listOf(PermissionRequest.RESOURCE_AUDIO_CAPTURE, PermissionRequest.RESOURCE_VIDEO_CAPTURE)
+        }.toTypedArray()
+        if (permissionsRequested.isEmpty()) {
+            request.deny() // no supported permissions were requested, so reject the request
+            logger.debug(
+                "($loggerTag) Denying permission - ${request.resources.joinToString()} are not supported"
+            )
+            return
+        }
+
+        if (checkSelfPermission(context, Manifest.permission.CAMERA) == PackageManager.PERMISSION_GRANTED) {
+            logger.debug("($loggerTag) Granting permission - permission already granted")
+            request.grant(permissionsRequested)
+        } else {
+            val isGranted = embeddedComponentManager.requestCameraPermission(context) ?: return
+            withContext(Dispatchers.Main) {
```

Comment:
If I'm reading the code correctly, either setting the dispatcher here isn't necessary, or we'd need to do so for the other `request.deny()` / `grant()` calls above (I think it's the former). Could you check?

Reply:
Agreed -- good point that it's safer to be explicit here